hbase-it/pom.xml

```diff
@@ -37,6 +37,11 @@
     <integrationtest.include>**/IntegrationTest*.java</integrationtest.include>
     <!-- To Run Tests with a particular Xmx Value use -Dfailsafe.Xmx=XXXg -->
     <failsafe.Xmx>4g</failsafe.Xmx>
+    <failsafe.jdk8.flags>-XX:+CMSClassUnloadingEnabled</failsafe.jdk8.flags>
```

Comment:
So this flag is permitted in JDK8 and JDK11 but not JDK17?

Reply:
Yes by default G1GC used in JDK11+ so only required for JDK8.